x/cdp/types/keys.go

```diff
@@ -10,6 +17,159 @@ const (
 	// RouterKey Top level router key
 	RouterKey = ModuleName
 
+	// QuerierRoute Top level query string
+	QuerierRoute = ModuleName
+
 	// DefaultParamspace default name for parameter store
 	DefaultParamspace = ModuleName
-)
\ No newline at end of file
+)
+
+var sep = []byte(":")
+
+// Keys for cdp store
+// Items are stored with the following key: values
+// - 0x00<cdpOwner_Bytes>: []cdpID
+//    - One cdp owner can control one cdp per collateral type
+// - 0x01<collateralDenomPrefix>:<cdpID_Bytes>: CDP
+//    - cdps are prefix by denom prefix so we can iterate over cdps of one type
+//    - uses : as separator, otherwise the cdp with ID 11 would be selected when iterating over denom with prefix 1
```

Comment:
Pretty sure separators aren't needed. If each part of a key is a known fixed length so there's no need to rely on a special `:` byte to seprate them. The collateralDenomPrefix is fixed at one byte right?
Maybe just update comment, can't see any harm in leaving separators in.